compiler/src/iree/compiler/Codegen/Common/IREEComprehensiveBufferizePass.cpp

```diff
@@ -241,6 +260,19 @@ void IREEComprehensiveBufferizePass::runOnOperation() {
   }
 }
 
+/// Run constant bufferize.
```

Comment:
nit: delete the comment. It is quite meaningless to me, and `runOnOperation` is very common in MLIR-based projects IMO.